MediaBrowser.Common/Json/Converters/JsonGuidConverter.cs

```diff
@@ -13,21 +13,13 @@ public class JsonGuidConverter : JsonConverter<Guid>
         public override Guid Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
         {
             var guidStr = reader.GetString();
-
             return guidStr == null ? Guid.Empty : new Guid(guidStr);
         }
 
         /// <inheritdoc />
         public override void Write(Utf8JsonWriter writer, Guid value, JsonSerializerOptions options)
         {
-            if (value == Guid.Empty)
-            {
-                writer.WriteNullValue();
```

Comment:
Why don't we want this anymore?

Reply:
This causes issues with the generated OpenApi spec, where the GUID in the spec is not-null, but null is returned.